app/ante/eip712.go

```diff
@@ -52,9 +52,9 @@ func init() {
 // transactions, as defined by the presence of an ExtensionOptionsWeb3Tx extension.
 func NewLegacyCosmosAnteHandlerEip712(options HandlerOptions) sdk.AnteHandler {
 	return sdk.ChainAnteDecorators(
-		RejectMessagesDecorator{}, // reject MsgEthereumTxs
+		NewRejectMessagesDecorator(options.DisabledCosmosMsgs),
```

Comment:
Is it mean that `vestingtypes.MsgCreateVestingAccount` will also be rejected by RejectMessagesDecorator?